st2client/st2client/base.py

```diff
@@ -253,12 +260,16 @@ def _get_cached_auth_token(self, client, username, password):
         file_st_mode = oct(os.stat(cached_token_path).st_mode & 0o777)
         others_st_mode = int(file_st_mode[-1])
 
-        if others_st_mode >= 4:
+        if others_st_mode >= 2:
             # Every user has access to this file which is dangerous
-            message = ('Permissions (%s) for cached token file "%s" are to permissive. Please '
+            message = ('Permissions (%s) for cached token file "%s" are too permissive. Please '
                        'restrict the permissions and make sure only your own user can read '
-                       'from the file' % (file_st_mode, cached_token_path))
-            self.LOG.warn(message)
+                       'from or write to the file.'
+                       '\n\n'
```

Comment:
NIT: I think that's too much new lines here which makes it pretty inconsistent with existing stackstorm warning messages by taking a lot of space. Example:

```
vagrant@stackstorm:~$ st2 action list
2018-06-26 14:50:42,463  WARNING - The StackStorm configuration directory permissions are insecure (too permissive).

You can fix this by running:

    chmod 770 /home/vagrant/.st2

2018-06-26 14:50:42,463  WARNING - The StackStorm configuration file permissions are insecure.

You can fix this by running:

    chmod 660 /home/vagrant/.st2/config

2018-06-26 14:50:42,463  WARNING - Permissions (0664) for cached token file "/home/vagrant/.st2/token-st2admin" are too permissive. Please restrict the permissions and make sure only your own user can read from or write to the file.

You can fix this by running:

    chmod o-rw /home/vagrant/.st2/token-st2admin
```

Reply:
Small nitpick - In some places in the log messages we use numerical permission levels and in other we use symbolic notation.

It's probably a good idea to unify it and use numerical / symbolic everywhere :)

Reply:
I'll simply remove the help text. I just usually like to give users some idea of how to fix a problem, which requires readable formatting, but it might not be the best for log messages.

The octal vs symbolic issue is more than just a notation difference, it's a functional one. The octal notation sets the permissions for the user, group, and "others", but the symbolic notation here performs operations on the permissions for just "others" and leaves the bits for the user and group intact.